doc/dev_guide/internals/iproto/keys.rst

```diff
@@ -50,6 +52,10 @@ General
             -   0x00 |br| MP_UINT
             -   Request type or response type
 
+        *   -   :ref:`IPROTO_UNKNOWN <internals-iproto-keys-unknown>`
+            -   -1 |br| MP_UINT
+            -   Unknown request or response type
```

Comment:
```suggestion
- Unknown request or response type
```

Actually, this key is used only for IPROTO handler overriding, see tarantool/doc#3288. Would be nice to mention it here or not forget to come back later.